client/modules/accounts/components/passwordReset/updatePasswordOverlay.js

```diff
@@ -0,0 +1,165 @@
+import React, { Component, PropTypes } from "react";
+import classnames from "classnames";
+import {
+  Button,
+  TextField,
+  Translation
+} from "/imports/plugins/core/ui/client/components";
+
+class UpdatePasswordOverlay extends Component {
+  constructor(props) {
```

Comment:
`this.state = { something: something }` is the standard way of initializing state in a component, inside the constructor (ES6). Haven't seen much of `state = { something }` for state initialization.